packages/blocks/src/_common/components/tags/styles.ts

```diff
@@ -3,13 +3,18 @@ import { css, unsafeCSS } from 'lit';
 
 export const styles = css`
   affine-multi-tag-select {
-    position: absolute;
-    width: 100%;
+    position: fixed;
```

Comment:
I think we should not hard coded a `fixed` position, since the editor may be embedded in an container with dynamic offset (e.g., toolbars in AFFiNE). Also the `!important` style is something that should be avoided.

Reply:
@doodlewind  I agree with your point of view. However, I've noticed an issue where the position of the modal becomes incorrect during dynamic use. How do you think this problem could be best resolved?